### DISCUSSION TITLE
Actividad 3 - Grupo 14

### DISCUSSION THREAD

**Discussion by Lele3002** [frlp-utn-ingsoft/catedra#136] [Show and tell]
https://github.com/JuanbyGuada/vetsoft

**Comment by RodrigoJacznik** [comment]
Hola chicos. Antes que nada perdón por la demora.

Revise la app hasta el commit c44df0c2f71ee638bc193b9c3191011c6bb5913d.

El trabajo esta muy bien, tanto los commits como los nombres de los branches. 
Me gusto que hayan documentado en el README.

La actividad esta aprobada.
